x-pack/solutions/security/plugins/security_solution/public/data_view_manager/constants.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 export const DEFAULT_SECURITY_SOLUTION_DATA_VIEW_ID = 'security-solution-default';
+export const DEFAULT_SECURITY_SOLUTION_ALERT_DATA_VIEW_ID = 'security-solution-alert';
```

Comment:
can we reuse the common const defined in `x-pack/solutions/security/plugins/security_solution/common/constants.ts`? :)